### ISSUE TITLE
let user set any number in Clipboard Entries Cound

### ISSUE THREAD

**Issue by lopippo** [pvanek/qlipper#32]
I have used Klipper up to now. But I would like to fade away from KDE at the moment (exploring awesome as WM) and thus I would like to have qlipper working for me. I use Klipper as my second memory  and have set it to store very large numbers of items. Why do you limit the number of entries to such a low number ?
I have compiled qlipper on Debian 8.0 fine and set the value to 2000, let's see what's going on.

Thanks. 
Fiilppo

**Comment by palinek** [comment]
Is the menu presenting the items usable with such a huge value?

**Comment by agaida** [comment]
no, not really

**Comment by lopippo** [comment]
No, but there is in KDE a clever way to serialize all the items to the screens progressively, because one of the items is a "More" item that will serialize a new set of items for the clipboard history. The number of items actually shown at any given moment is limited by the height of the screen. I find myself often going back many "More" buttons back  in history.

**Comment by albaphysic** [comment]
>serialize all the items to the screens progressively, because one of the items is a "More" item that will serialize a new set of items for the clipboard history

How do you do this? 

A more button or scroll bar along with infinite clipboard items and search function would be amazing. [Bulletin](https://github.com/SergKolo/indicator-bulletin) is the only clipboard manager that I've seen that has all that, but sadly isn't compatible with KDE.

**Comment by palinek** [comment]
> A more button or scroll bar along with infinite clipboard items and search function would be amazing. 

Now, when you said it... something similar, that we have in `nm-tray` or `mainmenu` in `lxqt-panel` (scroll area in menu), can be done here...

**Comment by albaphysic** [comment]
Actually I was partially wrong, Bulletin doesn't have a scroll bar and shows limited items. But it does save infinite history to a file and has a pop up search box that can search it. Something like that would be very useful IMO.
